samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -752,7 +744,7 @@ The `nRF Cloud Provisioning Service`_ allows you to securely provision and onboa
 
 .. note::
 
-   This service is not compatible with devices that use the nRF9160, but only newer nRF91x1 devices such as the nRF9161.
+   This service is not compatible with devices that use the nRF9160, but only newer nRF91x1 devices such as the nRF9151 and nRF9161.
```

Comment:
@glarsennordic : Could you confirm whether  "nRF Cloud Provisioning Service Support" also work for the nRF9151?